internal/pkg/service/common/etcdop/watch_test.go

```diff
@@ -14,45 +15,76 @@ import (
 func TestTypedPrefix_Watch(t *testing.T) {
 	t.Parallel()
 
-	ctx := context.Background()
-	c := etcdhelper.ClientForTest(t)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
4. Upravil som test aby nebol blokujuci.

`defer cancel()` zabezpeci, ze `watcher` po teste skonci.
Inak by bezal dalej na pozadi, kym nedobehnu vsetky testy v pkg.
Co je zbytocne, ale ako nevadi to 
(kym netestujes v nejakom inom teste, ze ziadna goroutine nezostala neukoncena).